src/library/librarycontrol.cpp

```diff
@@ -38,6 +38,19 @@ LoadToGroupController::LoadToGroupController(LibraryControl* pParent, const QStr
             this,
             &LoadToGroupController::slotLoadToGroupAndPlay);
 
+#ifdef __STEM__
+    m_loadSelectedTrackStems =
+            std::make_unique<ControlPushButton>(ConfigKey(group, "load_selected_track_stems"));
+    connect(m_loadSelectedTrackStems.get(),
+            &ControlObject::valueChanged,
+            this,
+            [this](double value) {
+                if (value >= 0 && value <= 2 << mixxx::kMaxSupportedStems) {
+                    emit loadToGroup(m_group, static_cast<uint>(value), false);
+                }
+            });
+#endif
```

Comment:
I am still in doubt that this will ever get used. Let's remove this until we have a use case for this along with a mapping that actually makes use of it. We may set up a n experimental branch, that includes this CO and the other ideas we have discussed here, that we have a test build ready and not block mappers on this. What do you think? 
 ```suggestion
```

Reply:
This is something I'm actually planning to use in the S4 mapping. Due to performance reasons, loading the whole stem track to just play a track accapella or instrumental is very expansive. 
Often in my set, I would want to play a track instrumental, with the vocals of another track, this means having to load 8 stereo track when 2 would have been enough. This makes is particularly unusable as I will likely use a third, sometime a fourth deck.
As we are adding that feature on the UI, I think it is wrong to not allow that to be mapped - I certainly would find it a bummer if I have to pick my mouse and keyboard every time I want to go over that usecase.
What would be the downside to get this control in?

Reply:
makes 2 of us

Reply:
I can very much acknowledge the use case. I am just In doubt that exactly this implementation is particularly useful. 
But I might be wrong. So how about postponing this question to the mapping PR? Or do you want to outline some details here?

Reply:
This CO is the essence of this PR - and the use case is obviously there now!

I am therefore against removing this CO from this PR!

Since all the arguments have already been exchanged, I think it's time to start a vote of the Mixxx Core developers.

Reply:
I have added documentation of the CO in the description to help summarising the proposed behaviour